app/templates/profile.hbs

```diff
@@ -1,3 +1,13 @@
-{{#travis-layout layoutName="layouts/profile" class="profile-view"}}
-  {{outlet}}
-{{/travis-layout}}
+{{#if showLoading}}
```

Comment:
Is it possible to use the “loading substate” for this kind of thing instead?

Reply:
No, unfortunately, this kinda hack is necessary, because until the entire path is rendered the `router.currentRouteName` is different from `account.settings.unsubscribe` and there's no way to figure out if it's an unsubscribe route or not. This causes the profile layout to be rendered and only then replaced by `email-unsubscribe`. Thus, it's necessary to show loading screen until the entire path is resolved, not only the `profile` part.